pkgs/development/ocaml-modules/ocaml-protoc-plugin/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, fetchFromGitHub
+, buildDunePackage
+, pkg-config
+, protobuf
+, zarith
+, ppx_deriving
+, ppx_deriving_yojson
+, re
+, dune-site
+, ppx_expect
+}:
+
+buildDunePackage rec {
+  pname = "ocaml-protoc-plugin";
+  version = "4.3.1";
+
+  duneVersion = "3";
+
+  minimalOcamlVersion = "4.08";
+
+  src = fetchFromGitHub {
+    owner = "issuu";
+    repo = pname;
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21